hail/python/hailtop/utils/utils.py

```diff
@@ -111,12 +112,14 @@ def digits_needed(i: int) -> int:
     return 1 + digits_needed(i // 10)
 
 
-def grouped(n: int, ls: List[T]) -> Iterable[List[T]]:
+def grouped(n: int, ls: Iterable[T]) -> Iterable[List[T]]:
```

Comment:
Based on the above usage, shouldn't this be `Iterator`?

Reply:
We use `grouped` in other places where we pass a `List`. Those would be type failures if this was `Iterator[T]`.